WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -3067,12 +3109,31 @@ public void onTrackableEvent(TrackableEvent event, Map<String, String> propertie
     }
 
     @Override public void onStoryComposerLoadRequested(ArrayList<Object> mediaFiles, String blockId) {
+        if (mLoadStoryFromStoriesPrefsUseCase.anyMediaIdsInGutenbergStoryBlockAreCorrupt(mediaFiles)) {
```

Comment:
Could we rename this method to 
```suggestion
        if (mLoadStoryFromStoriesPrefsUseCase.areMediaIdsInGutenbergStoryBlockCorrupt(mediaFiles)) {
```

Reply:
Good suggestion - this method is going away in #13103 in commit b8d6c4f7ca81bfc2c55025f48173c70ee3434d0f 🙇 
I have a comment to do though:  I just wanted to know if _any_ mediaId was corrupt, not _all_, which might be implied in the suggested name, but maybe I'm just being picky anyway 😅 🙏 
Thanks!